docs/1.5/extensions/external-links.md

```diff
@@ -30,9 +31,12 @@ $environment->addExtension(new ExternalLinkExtension());
 // Set your configuration
 $config = [
     'external_link' => [
-        'internal_hosts' => 'www.example.com', // Don't forget to set this!
+        'internal_hosts' => 'www.example.com', // TODO: Don't forget to set this!
         'open_in_new_window' => true,
         'html_class' => 'external-link',
+        'nofollow' => 'external',
+        'noopener' => 'all',
```

Comment:
This should default to `external` as internal links may need this capability.

Reply:
It does, this just wasn't a good example to share :upside_down_face: